rustyms/src/shared/glycan.rs

```diff
@@ -663,58 +734,61 @@ pub enum GlycanSubstituent {
     Water,
 }
 
+impl GlycanSubstituent {
+    /// Get the symbol used to denote this substituent
+    pub const fn notation(self) -> &'static str {
+        match self {
+            Self::Acetimidoyl => "Am",
+            Self::Acetyl => "Ac",
+            Self::AcetylAlanyl => "Ala2Ac",
+            Self::AcetylGlutaminyl => "Gln2Ac",
+            Self::Acid => "A",
+            Self::Alanyl => "Ala",
+            Self::Alcohol => "ol",
+            Self::Amino => "N",
+            Self::Aric => "aric",
+            Self::CargoxyEthylidene => "Pyr",
+            Self::Deoxy => "d",
+            Self::Didehydro => "en",
+            Self::DiHydroxyButyryl => "3,4Hb",
+            Self::DiMethyl => "DiMe",
+            Self::DiMethylAcetimidoyl => "AmMe2",
+            Self::DiMethylGlyceryl => "Gr2,3Me2",
```

Comment:
The numerals here may refer to attachment sites on the carbon backbone, not part of the substituent themselves.

Reply:
Thanks, I also just realised these should just be stored as a `Dimethyl` and a `Acetimidoyl` separatedly, so I will update that.